sort-selection/semicolon.js

```diff
@@ -1,53 +0,0 @@
-const vscode = require('vscode');
-
-function activate(context) 
-{
-	console.log('"sort-selection-semicolon" is active!');
```

Comment:
Are you sure this file was intended to be removed? it looks like it provided the `sort-selection-semicolon` functionality.

By the way, my previous comment was that most of the functions are duplicated, the only differences are L7, L5 and L50. So I suggested to make a third file for the common code, and keep just the differences in these two files.

Reply:
Yes, I created another extension for semicolon, that is mostly used for Linux desktop files like [BCompare desktop file](https://github.com/gusbemacbe/wine-desktop-files/blob/master/bcompare/bcompare.desktop). My goal is trying to unifying the comma and semicolon in the same only file `extension.js`